<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prototype</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract data</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>421</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="brainLabel">
         <property name="text">
          <string>No EPI-images chosen</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="stimuliLabel">
         <property name="text">
          <string>No stimuli chosen</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="maskLabel">
         <property name="text">
          <string>No mask chosen</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="brainButton">
         <property name="text">
          <string>Choose EPI-images</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stimuliButton">
         <property name="text">
          <string>Choose stimuli</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="maskButton">
         <property name="text">
          <string>Choose mask</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="add_individual_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add individual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_group_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Add group</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="tree_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Pick SPM Path</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
